addon/styles/addon.scss

```diff
@@ -132,3 +132,9 @@
     }
   }
 }
+
+.ember-tooltip-or-popover-target-short-circuit {
```

Comment:
TODO: Remove this in favor of the [absolute-parent-fix](https://github.com/sir-dunxalot/ember-tooltips/tree/absolute-parent-fix) branch.